### ISSUE TITLE
Application should be accessible without the need for a mouse

### ISSUE THREAD

**Issue by rlueder** [rlueder/tidytweets#33]
Users should be able to navigate by keyboard only, make sure aria labels are used properly and consistently.

https://developer.mozilla.org/en-US/docs/Web/Accessibility
https://reactjs.org/docs/accessibility.html#testing-accessibility-in-the-browser

**Comment by Ilnicki010** [comment]
Great point! Maybe we should add some automated a11y tests to the pipeline

**Comment by Ilnicki010** [comment]
I'm into a11y, I'd glad to take care of that

**Comment by rlueder** [comment]
Sounds good! Let me know if you need any help with the test framework.
